build/deploy/examples/minimum.jsonnet

```diff
@@ -3,19 +3,20 @@ local dss = import '../../deploy/dss.libsonnet';
 local metadataBase = import '../../deploy/metadata_base.libsonnet';
 
 local metadata = metadataBase {
-  namespace: 'dss-main',
+  namespace: 'namespace',
```

Comment:
The default namespace argument value in make-certs.py is 'dss-main' -- the value here should match if the namespace argument is omitted from make-certs.  We should probably also add a callout comment that this should match whatever was provided to make-certs and apply-certs, if that is true.

Reply:
done.